### ISSUE TITLE
Router won't upgrade from 18.7.10_4 to 19.1 (or 19.1.1)

### ISSUE THREAD

**Issue by kuerious** [opnsense/core#3210]
Cross-posting from the forums.

I started at 18.7.10_4, OpenSSL edition on the Protectli FW4A w/ 8GB:
https://i.imgur.com/GxDdDg2.png

From there, back at the release of 19.1 (not the now-changed 1191.1), I followed **ALL** of the steps listed in order. Then I scoured the forums, and tried many different things. And I even took screenshots as I went along.

* Attempted to use the "Update" instead of the "Upgrade" button, makes no difference:
https://i.imgur.com/urtUglm.png

* Sometimes I'd get this message:
https://i.imgur.com/GNpTUU3.png

* Other times, I'd get this one:
https://i.imgur.com/ZsFO86I.png

* The system rebooted after [i]actually showing it downloading[/i] the 19.1 images & kernel:
https://i.imgur.com/CE2eHvV.png

* No matter what, trying an upgrade [i]looked[/i] like it was going to work, using "Unlock" and "Upgrade":
https://i.imgur.com/Wf5ebgx.png

Steps taken:
- Changed mirror from "default" to "Leaseweb", ran "updates", it DL'ed 18.7.10_4 ([i]again[/i], and rebooted:
**Note: lost this screenshot**

- Considered changing from "default" (currently "OpenSSL") over to "LibreSSL" but wasn't sure if it was HW-specific or just a choice:
https://i.imgur.com/XtyXW9s.png

I looked through all of the logs that made logical sense, looking for "fail"/"failure"/"warning" and all the other ones...; I tried updating from the console/CLI (using both "Y" and "19.1"); I've dug through so many forum posts until I fell asleep at my keyboard. I still haven't found the "right" or "wrong" thing I'm doing/not-doing so as to make this work.

I'm not really ready to take down the internet for half of a day just to reinstall my whole install. And while I have an ESXi server, it'll still take half of a day to get a new VM up and then re-route the networking.

Does ANYONE know what's going on or causing this problem?

**Comment by kuerious** [comment]
No answers. No suggestions.

So, I _did_ take down the internet. Did a fresh, FULL install of 19.1 ... and I STILL CANNOT INSTALL 19.1.1

<shrug> ... I have no idea.

**Comment by fichtner** [comment]
Unfortunately, it all looks normal. There's no "sometimes, other times" as far as I can see as you're showing the major upgrade procedure with its warnings and confirmation dialogs.

First picture, however, already indicates your install is clobbered in some way:

While you are on 18.7.10_4, it says 11.2-RELEASE-p8-HBSD which is 19.1.

So your packages upgrades are blocking.

Did you install Sensei, Saltstack, use the FreeBSD package repo or built ports manually using the ports tree?


Cheers,
Franco

**Comment by kuerious** [comment]
Okay, follow-up. A reformat, initial install, reboot, restore of backup, and reboot DID allow 19.1 to install. While update to 19.1.1 initially appeared to NOT work, another reboot returned to 19.1 screen. This 2nd time, it showed the correct update files for 19.1.1. Ran the update, allowed a reboot, and it appears that I am running 19.1.1 just fine.

While, yes, I did perform a Sensei install prior to the 19.1.x upgrade, I had initially disabled all services to allow for no problems during the upgrade. When it failed, I ran the script to remove Sensei, rebooted, and tried again.

Now that 19.1.1 is running with all updates, I've installed Sensei again according to the site's instructions. It has caused at least one crash (see: https://i.imgur.com/1MlsZzt.png ). A reboot restored service, and I am monitoring the services (and CPU/Mem/SSD usage) in case it recurs. System specs:
* Protectli FW4A incl./ AESNI
* Intel Atom quad-core @1.91GHz
* mSATA 120GB SSD
* RAM 8GB PC3L-14900
